test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ControllerActionInvokerTest.cs

```diff
@@ -2091,7 +2091,7 @@ public void CreateActionResult_ReturnsObjectContentResult(Type type, object inpu
 
             var binder = new Mock<IModelBinder>();
```

Comment:
Or did you get rid of `new Mock<IModelBinder>()` primarily where the setup was a bit more complex?